api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionIrsPaths.scala

```diff
@@ -28,16 +34,14 @@ trait SubmissionIrsPaths
   implicit val timeout: Timeout
 
   // institutions/<institutionId>/filings/<period>/submissions/<submissionId>/irs
-  // NOTE:  This is currently a mocked, static endpoint
   def submissionIrsPath(institutionId: String)(implicit ec: ExecutionContext) =
     path("filings" / Segment / "submissions" / IntNumber / "irs") { (period, submissionId) =>
-      timedGet { uri =>
-        val supervisor = system.actorSelection("/user/supervisor")
+      timedGet { _ =>
+        val larTotalRepository = new LarTotalRepository(config)
 
-        //To avoid having to deal with relative paths on different systems
-        val irsJson = "{\n  \"msas\": [\n    {\n      \"id\": \"123\",\n      \"name\": \"MSA 123\",\n      \"totalLARS\": 4,\n      \"totalAmount\": 123,\n      \"conv\": 4,\n      \"FHA\": 0,\n      \"VA\": 0,\n      \"FSA\": 0,\n      \"1to4Family\": 4,\n      \"MFD\": 0,\n      \"multiFamily\": 0,\n      \"homePurchase\": 0,\n      \"homeImprovement\": 0,\n      \"refinance\": 4\n    },\n    {\n      \"id\": \"456\",\n      \"name\": \"MSA 456\",\n      \"totalLARS\": 5,\n      \"totalAmount\": 456,\n      \"conv\": 5,\n      \"FHA\": 0,\n      \"VA\": 0,\n      \"FSA\": 0,\n      \"1to4Family\": 5,\n      \"MFD\": 0,\n      \"multiFamily\": 0,\n      \"homePurchase\": 0,\n      \"homeImprovement\": 0,\n      \"refinance\": 5\n    }\n  ],\n   \"status\": {\n       \"code\": 10,\n       \"message\": \"IRS report generated\"\n     }}"
+        val data = larTotalRepository.getMsaSource().map(x => ChunkStreamPart(x.toString))
 
-        val response = HttpResponse(StatusCodes.OK, entity = HttpEntity(ContentTypes.`application/json`, irsJson))
+        val response = HttpResponse(StatusCodes.OK, entity = HttpEntity.Chunked(ContentTypes.`application/json`, data))
 
```

Comment:
I think the `lars_total` view is not being created (happened to me when I ran it, not sure if that behavior is consistent)